test/distributed/_sharded_tensor/test_sharded_tensor.py

```diff
@@ -219,6 +257,36 @@ def test_complete_world_size(self):
                     else:
                         self.assertEqual((3, 20), shard.tensor.size())
 
+
+    @with_comms
+    @skip_if_lt_x_gpu(4)
+    @requires_nccl()
+    def test_create_sharded_tensor_with_ones(self):
+        """ Test _sharded_tensor.ones(...) """
+
+        spec = ChunkShardingSpec(
+            dim=0,
+            placements=[
+                "rank:0/cuda:0",
+                "rank:1/cuda:1",
+                "rank:2/cuda:2",
+                "rank:3/cuda:3",
+            ],
+        )
+        sharded_tensor = _sharded_tensor.ones(spec, 10, 20)
```

Comment:
nit: store the shape in a variable and check the shards by dividing/modding for the appropriate rank? 

i.e. 
```
h, w = 10, 20
tensor = sharded.ones(spec, h, w)
...
self.assertEqual(local_shard, ones(ceil(h//world_size, w)

```

Reply:
Good point, updated per suggested